testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -270,7 +270,8 @@ private static String createQueryDebuggingSummary(BasicQueryInfo basicQueryInfo,
         }
     }
 
-    @Test
+    // TODO @Test - Temporarily disabled to avoid test classes running twice. Re-enable once all tests migrated to JUnit.
```

Comment:
issue?

Reply:
we will remember -- when cleaning classpath from testng-related stuff